lib/components/term.js

```diff
@@ -234,12 +292,20 @@ export default class Term extends React.PureComponent {
     return !e.catched;
   }
 
-  componentWillReceiveProps(nextProps) {
+  async componentWillReceiveProps(nextProps) {
     if (!this.props.cleared && nextProps.cleared) {
       this.clear();
     }
+
     const nextTermOptions = getTermOptions(nextProps);
 
+    if (nextProps.bellSoundURL) {
+      const value = await getFileData(nextProps.bellSoundURL);
```

Comment:
I think it should not be done here.
It will be done each time that you open a new pane/tab/window.

Maybe better to do this in the main process but I don't know how to do this fetch asynchronously. Maybe by reemitting a `config change` RPC event when done, to dispatch a new `CONFIG_RELOAD` redux action in each window.

It will maybe require adding some caching mechanism when decorating configuration.